docs/en/ingest-management/fleet-agent-proxy-support.asciidoc

```diff
@@ -0,0 +1,250 @@
+[[fleet-agent-proxy-support]]
+= Use a proxy server with {agent} and {fleet}
+
+Your organization’s security strategy and other considerations may require you
+to use a proxy server between some components in your deployment. Support is
```

Comment:
For example, you may have a firewall rule that prevents endpoints from connecting directly to Elasticsearch. In this scenario, you'd have the Elastic Agent connect to a proxy, and the proxy connects to Elasticsearch through the firewall.